dashboard/config/grafana/dashboards/psm/cxx_experiments.json

```diff
@@ -2921,7 +2921,7 @@
     },
     "timepicker": {},
     "timezone": "browser",
-    "title": "gRPC C++ Experiments: Performance Dashboard",
-    "uid": "hork-20230731",
+    "title": "gRPC-Core Experiments - Performance Dashboard",
+    "uid": "cxx_experiments",
```

Comment:
Nit: If this goes in a url, perhaps dash would make more sense than underscore.

Reply:
Done